src/app/app.component.html

```diff
@@ -8,13 +8,29 @@ <h1>
 <h2>Here are some links to help you start: </h2>
 <ul>
   <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
+    <h2>
```

Comment:
you shouldn't use `<h2 />` tag here, pick `<h3 />` or skip such element at all since it creates invalid document outline